packages/ckeditor5-minimap/src/minimapiframeview.ts

```diff
@@ -75,38 +70,32 @@ export default class MinimapIframeView extends IframeView {
 	/**
 	 * @inheritDoc
 	 */
-	render() {
+	public override render(): Promise<unknown> {
 		return super.render().then( () => {
 			this._prepareDocument();
 		} );
 	}
 
 	/**
 	 * Sets the new height of the iframe.
-	 *
-	 * @param {Number} newHeight
 	 */
-	setHeight( newHeight ) {
+	public setHeight( newHeight: number ): void {
 		this.height = newHeight;
 	}
 
 	/**
 	 * Sets the top offset of the iframe to move it around vertically.
-	 *
-	 * @param {Number} newOffset
 	 */
-	setTopOffset( newOffset ) {
+	public setTopOffset( newOffset: number ): void {
 		this.top = newOffset;
 	}
 
 	/**
 	 * Sets the internal structure of the `<iframe>` readying it to display the
 	 * minimap element.
-	 *
-	 * @private
 	 */
-	_prepareDocument() {
-		const iframeDocument = this.element.contentWindow.document;
+	private _prepareDocument(): void {
+		const iframeDocument = ( this.element as any ).contentWindow.document;
```

Comment:
Change `IframeView` to extend `View<HTMLIFrameElement>` instead of `View` and the cast to any won't be needed.